src/ResourceManagement/Resource/ResourceManagement/Generated/Models/DeploymentValidateResponse.cs

```diff
@@ -31,12 +31,12 @@ namespace Microsoft.Azure.Management.Resources.Models
     /// </summary>
     public partial class DeploymentValidateResponse : AzureOperationResponse
     {
-        private ResourceManagementErrorWithDetails _error;
+        private ResourceManagementError _error;
         
         /// <summary>
         /// Optional. Gets or sets validation error.
         /// </summary>
-        public ResourceManagementErrorWithDetails Error
+        public ResourceManagementError Error
```

Comment:
@vivsriaus This is a breaking API change.  I thought you wanted to avoid breaks?